app/src/androidTest/java/org/mozilla/tv/firefox/ui/screenshots/SettingsTest.java

```diff
@@ -0,0 +1,75 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ui.screenshots;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.uiautomator.UiDevice;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mozilla.tv.firefox.MainActivity;
+import org.mozilla.tv.firefox.R;
+import org.mozilla.tv.firefox.helpers.MainActivityTestRule;
+
+import tools.fastlane.screengrab.Screengrab;
+import tools.fastlane.screengrab.locale.LocaleTestRule;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+
+public class SettingsTest extends ScreenshotTest {
+
+    private UiDevice mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+
+    @ClassRule
```

Comment:
Oh! did this not work with a kotlin file?

Reply:
No it did not - it exhibited two issues,
- It did not run on anything other than EN-US locale.  (i.e. locale could not be switched via fastlane)
- It took duplicate screenshots (two of every kind)

Reply:
Ah, I guess I originally patterned this off of PocketRecommendationTest but that's ignored, so maybe that test didn't work either! Optionally could add a comment for that test (which is also a kotlin test)

Reply:
ah, should have read your comments before I merged... that's right, I should have converted them to java before ignoring them, but that was ideal world scenario :)